### DISCUSSION TITLE
Performance of 86Box vs PCem on Pentium II

### DISCUSSION THREAD

**Discussion by silverlays** [86Box/86Box#3117] [Q&A] [answered]
Hi everyone,

I am actually trying the two x86 emulator I have found, but that question bothers me: 

**Is that normal that 86box as a lack of performance in comparaison of PCem v17 ?**
_(I'm running on Ryzen 5 5600x (6C/12T) and trying to emulate Pentium II 350Mhz)_

With PCem, I have almost constantly 100% of running speed, but on 86Box (ODR and NDR) I always had drop, especially when sounds are played (on YMFM).

I really would like to have your feedbacks on this. Thanks in advance

**Comment by jriwanek** [comment, marked as answer]
Have you set the HDD speed to something other than RAM Disk in 86box, also, have you tried ymfm instead of nukedopl for sound?

It's worth remembering that 86box has a greater focus on emulation *accuracy*, which means it can be heavier on the host than pcem.

It's also worth noting that we have a greater emphasis on maintaining the project and fixing bugs, so performance shouldn't be the only variable in consideration too.

After all, doesn't matter if x is faster than y if it doesn't work as well for what you need it to.

Another important set of questions:
1) Are you comparing pcem with New dynarec 86box or old dynarec? (Pcem uses New Dynarec) (edit: Question already answered above)
2) Are you comparing a 32 bit build of pcem with a 64 bit build of 86box? The 64 bit builds tend to be a little slower than the 32 bit builds (And tend to be more buggy too).

**Reply by silverlays** [reply]
To start, thanks for your quick answer. There is my answers:

> Have you set the HDD speed to something other than RAM Disk in 86box, also, have you tried ymfm instead of nukedopl for sound?

**Yes, HDD are configured with 7200RPM, and yes, I use YMFM**

> Are you comparing pcem with New dynarec 86box or old dynarec? (Pcem uses New Dynarec)

**I have tested both ODR and NDR, on 32 and 64 bits**
<br/>
_To finish, it been a while that I am using 86box, it works like a charm as long as I don't try to go over pentium I at 200Mhz. In addition, I would like to say that I'm not here to blame the 86Box team or whatever, I just try to understand, and if possible, find a solution. I love 86box and what it brings in terms of quality of life compared precisely to PCem_

Thanks you

**Reply by Efflixi86Box** [reply]
I would like to add that I have a cpu very comparable to yours in single threaded performance (which is what 86box cares about) and the fastest I can emulate (tested on hundreds of configs) is a P2 266 in DOS or  p2 233 in Win9x or alternatively, a mobile pentium 300mmx (super socket 7). Extrapolating out performance figures, there are only a couple consumer CPU's capable of running a p2 350.

**Reply by robman501a** [reply]
I also have the Ryzen 5600x and I don't think this CPU can handle that speed of a P2. Well, if you want a "fast" computer (like an Pentium 4), I suppose you could use VirtualBox as that passes your host CPU over to the guest but at you can't customize the hardware. 
That being said... that's the only way I can run Windows XP with the Ryzen 5600x. HAHA!
